easel/lib/screens/choose_format_screen.dart

```diff
@@ -64,11 +64,29 @@ class _ChooseFormatScreenState extends State<ChooseFormatScreen> {
             )));
   }
 
-  void showErrorDialog({NFTTypes? type}) {
+  String getFileFormatNotSupportedMsg({required NftFormat nftFormat, required PickedFileModel nftFile}) {
+    final fileName = nftFile.fileName.replaceAll(".${nftFile.extension}", "");
+    String formatMustBe = "";
+    if (nftFormat.format == NFTTypes.image) {
+      formatMustBe = LocaleKeys.image_nft_must_be.tr();
+    } else if (nftFormat.format == NFTTypes.audio) {
+      formatMustBe = LocaleKeys.audio_nft_must_be.tr();
+    } else if (nftFormat.format == NFTTypes.video) {
+      formatMustBe = LocaleKeys.video_nft_must_be.tr();
+    } else if (nftFormat.format == NFTTypes.threeD) {
+      formatMustBe = LocaleKeys.three_d_nft_must_be.tr();
+    } else if (nftFormat.format == NFTTypes.pdf) {
+      formatMustBe = LocaleKeys.pdf_nft_must_be.tr();
+    }
+    return '"$fileName" ${LocaleKeys.could_not_uploaded.tr()} $formatMustBe';
```

Comment:
Not good approach use argumented translations.

    return '"$fileName" ${LocaleKeys.could_not_uploaded.tr()} $formatMustBe';